Makefile

```diff
@@ -282,12 +288,16 @@ $(BOOTFW_SRC):
 
 BOOTFW := \
 	$(BOOTFWDIR)/bcm2710-rpi-3-b.dtb \
+	$(BOOTFWDIR)/bcm2710-rpi-3-b-plus.dtb \
 	$(BOOTFWDIR)/bcm2710-rpi-cm3.dtb \
 	$(BOOTFWDIR)/bootcode.bin \
 	$(BOOTFWDIR)/start.elf \
 	$(BOOTFWDIR)/start4.elf \
 	$(BOOTFWDIR)/fixup.dat \
 	$(BOOTFWDIR)/fixup4.dat
+ifeq ($(ARCH:arm64=),)
+BOOTFW := $(BOOTFWDIR)/bcm2711-rpi-4-b.dtb $(BOOTFW)
```

Comment:
Should start4.elf and fixup4.dat go in this ifeq section as well?

Reply:
good point, since we don't plan to support armhf on rpi4.